runtime/cmake/platform/os/zos.cmake

```diff
@@ -25,7 +25,9 @@ set(CMAKE_SHARED_LIBRARY_PREFIX "lib")
 
 set(CMAKE_IMPORT_LIBRARY_PREFIX "lib")
 
-list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
-    "\"-Wc,inline(auto,noreport,600,5000)\""
-    "\"-Wc,list(),offset,gonumber\""
-)
+if(${OMR_TOOLCONFIG} STREQUAL "xlc")
+	list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
+		"\"-Wc,inline(auto,noreport,600,5000)\""
+		"\"-Wc,list(),offset,gonumber\""
+	)
+endif()
```

Comment:
Perhaps these options belong in `xlc.cmake`?
Are there equivalent options for OpenXL?

Reply:
As mentioned above with Peters intial comment, a issue has been created: https://github.ibm.com/runtimes/openj9-openjdk-jdk21-zos/issues/320 ...
There are no current exact equivalents, and the cpp team has suggested to not put equivalent optimization flags as there is no guarantee it will benefit or worsen the performance.

I can move this to `xlc.cmake` though if thats better ...

Reply:
Moved into `xlc.cmake` in https://github.com/eclipse-openj9/openj9/commit/1be349dbb7cbfe8c1a429e852e0b71f08b35489f